cpp/src/barretenberg/plonk/proof_system/prover/c_bind.cpp

```diff
@@ -48,8 +47,8 @@ WASM_EXPORT void* test_env_load_prover_crs(size_t num_points)
 {
     return env_load_prover_crs(num_points);
 }
-typedef std::conditional_t<plonk::SYSTEM_COMPOSER == ComposerType::TURBO, plonk::TurboProver, plonk::UltraProver>
-    WasmProver;
+
+using WasmProver = plonk::UltraProver;
```

Comment:
We are not yet using the WasmProver in A3, but we might in the future. Does this hardcode the WasmProver to UltraPlonk? Do we still want the ability to use it with Turbo? Does the Noir team need the ability to use these bindings with Turbo if a user chooses to use Turbo instead of Ultra @phated?

Reply:
We currently don't use this file at all, but it looks like it's hardcoding the prover

Reply:
OK, good to know, so if @ludamad is happy then it's probably fine to fix this. As for Noir, there is no Turbo verifier smart contract atm, so I think there is no short-term need to switch to Turbo.

Reply:
I'm not clear when we might want to use WasmProver so not sure I have context

Reply:
If the Noir team isn't using it and doesn't plan to, then my intuition is that it won't be important to us either. I think we should try to use a similar proving process as the Noir team from a3 TS.

So this diff is probably fine.

Reply:
We don't use it at the moment but @TomAFrench might have some more insight on this as there were discussions about passing a pointer to the prover in order to enable multithreading with wasm. For now though we do not need the ability to use these bindings with Turbo and I think diff is fine as is.